### ISSUE TITLE
Add screenshots to Readme

### ISSUE THREAD

**Issue by stackotter** [flowkey/UIKit-cross-platform#356]
It would be great if developers checking out the project could easily find screenshots of how the demo app currently looks running on Android. And in general, screenshots are a great way to create a more engaging readme.

**Comment by DanBurkhardt** [comment]
@stackotter agreed, but you should be able to issue a PR with this if you get a few mins.

**Comment by stackotter** [comment]
I don't have Android Studio installed, but I'll submit a PR if I ever get around to doing that and running the demo in the emulator
